### PR TITLE
Add manifest for org.kde.kfind

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://webchat.kde.org/#/room/#flatpak:kde.org
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfind

File: org.kde.kfind.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "id": "org.kde.kfind",
+    "branch": "master",
+    "rename-icon": "kfind",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfind",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
```
Comment (hfiguiere) [inline org.kde.kfind.json:9]:
```suggestion
    "finish-args": [
        "--share=ipc", 
        "--socket=fallback-x11", 
        "--socket=wayland", 
        "--filesystem=host" 
    ],
```

**Comment by nedrichards** [comment]
@flathub/kde FYI

File: org.kde.kfind.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    "modules": [
+        {
+            "name": "kfind",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "260fec641fa8a596144e8bb3cbfbab92fb9034df",
```
Comment (tsdgeos) [inline org.kde.kfind.json:17]: you need to use a tarball not a git commit

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfind

File: org.kde.kfind.json (hunk 1 above)
Comment (flyingcakes85) [reply org.kde.kfind.json]: Thanks, fixed in latest commit.

File: org.kde.kfind.json (hunk 2 above)
Comment (flyingcakes85) [reply org.kde.kfind.json]: Thanks, the latest commit now uses tarball.

File: org.kde.kfind.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.kfind",
+    "branch": "master",
```
Comment (barthalion) [inline org.kde.kfind.json:3]:
```suggestion
```

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfind

**Comment by travier** [comment]
```
libEGL warning: wayland-egl: could not open /dev/dri/renderD128 (No such file or directory)
```

Needs `--device=dri` in finish-args.

I think we can also remove the `share/man` directory.

Apart from that LGTM.

**Comment by flyingcakes85** [comment]
bot, build org.kde.kfind

**Pull request closed without merging**